ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/TopBarComposable.kt

```diff
@@ -44,11 +46,20 @@ fun TopBar(
             .heightIn(max = 24.dp)
         )
       } else {
-        ActionButton(
-          imagePainter = painterResource(R.drawable.ic_arrow_back),
-          description = "Back",
-          onClick = onClickBack
-        )
+        if (onClickBack != null)
+          ActionButton(
+            imagePainter = painterResource(R.drawable.ic_arrow_back),
+            description = "Back",
+            onClick = onClickBack
+          )
+        else {
+          val activity = LocalContext.current as? Activity
+          ActionButton(
+            imagePainter = painterResource(R.drawable.ic_arrow_back),
+            description = "Back",
+            onClick = { activity?.onBackPressed() }
+          )
+        }
```

Comment:
you could reduce this code doing this:

          ActionButton(
            imagePainter = painterResource(R.drawable.ic_arrow_back),
            description = "Back",
            onClick =  onClickBack ?: activity?.onBackPressed()
          )